packages/nextjs-mf/src/plugins/ContainerStatsPlugin.js

```diff
@@ -1,58 +1,59 @@
-const fs = require('fs');
 const path = require('path');
 
-
-
 class ModuleChunkDependencyPlugin {
```

Comment:
Might delete this plugin. Module Hoisting does a good job.
Some of this plugin could be salvaged to improve perf of chunk correlation 
Overall, this is currently not used.